src/lib/convert-forms.js

```diff
@@ -4,6 +4,7 @@ const argsFormFilter = require('./args-form-filter');
 const exec = require('./exec-promise');
 const fs = require('./sync-fs');
 const { getFormDir } = require('./forms-utils');
+const { escapeWhitespacesInPath } = require('./forms-utils');
```

Comment:
is possible to do this `const { getFormDir , escapeWhitespacesInPath } = require('./forms-utils')` here?

Reply:
Oh, great one! thanks, just applied it